packages/core/src/modules/proofs/protocol/v1/__tests__/indy-proof-presentation.test.ts

```diff
@@ -1,28 +1,28 @@
 import type { Agent } from '../../../../../agent/Agent'
 import type { ConnectionRecord } from '../../../../connections/repository/ConnectionRecord'
+import type { ProofExchangeRecord } from '../../../repository'
 import type { PresentationPreview } from '../models/V1PresentationPreview'
 
-import { setupProofsTest, waitForProofExchangeRecord } from '../../../../../../tests/helpers'
+import { setupIndyProofsTest, waitForProofExchangeRecord } from '../../../../../../tests/helpers'
```

Comment:
setupIndyProofsTest was renamed recently to setupProofsTest. I think this PR is undoing a previous one.

Reply:
There are now two setupProofs methods in the helpers file. One is called setupProofsTest and one is called setupIndyProofsTest.

This is a bit confusing, so I have renamed setupProofsTest to now be setupJsonLdProofsTest